### DISCUSSION TITLE
Sync Error: Operation Error: Could not move temporary file

### DISCUSSION THREAD

**Discussion by shadowofdeepdark** [windingwind/zotero-better-notes#1048] [Q&A]
The problem: ![image](https://github.com/user-attachments/assets/e3fc086b-5416-45df-a6d0-0132af466f0f)
Zotero Edition: 7.0.0-beta.105+c5c613c82 (64-bit)
Set up: ![image](https://github.com/user-attachments/assets/e23314e2-24ae-4166-956a-40e72aae3e44)
Context: trying to sync with Obsidian and the file was saved in the location of attempted obsidian file

**Comment by windingwind** [comment]
Please report as an [issue](https://github.com/windingwind/zotero-better-notes/issues).

**Reply by shadowofdeepdark** [reply]
Thanks! Just did:)
